.eslintrc.js

```diff
@@ -0,0 +1,205 @@
+module.exports = {
+	'root': true,
+    'env': {},
+    'extends': [
+        'plugin:@typescript-eslint/recommended',
+        'plugin:@typescript-eslint/recommended-requiring-type-checking'
+    ],
+    'parser': '@typescript-eslint/parser',
+    'parserOptions': {
+        'project': 'tsconfig.eslint.json',
+        'sourceType': 'module'
+    },
+    'plugins': [
+        '@typescript-eslint',
+        '@typescript-eslint/tslint'
+    ],
+    'rules': {
+        '@typescript-eslint/adjacent-overload-signatures': 'error',
+        '@typescript-eslint/array-type': ['error', {'default': 'array-simple'}],
+        '@typescript-eslint/ban-types': 'error',
```

Comment:
here should be an array according to docs.

Reply:
With a string 'error' it's using default settings. Honestly, I did not set up each rule individually. There is a tool that converts TSLint config into ESLint config - I used it and only adjusted the rules that caused problems with our code styling.